Blog week 9 2020 (#109)
packit-service/packit.dev: content/posts/February-2020.md

[(#715)]: https://github.com/packit-service/packit/pull/715
[(#406)]: https://github.com/packit-service/packit-service/pull/406
[(#420)]: https://github.com/packit-service/packit-service/pull/420

## Week 9

> Dist-git patches are applied with `-p1` [(#730)]. Packit specfile code was improved [(#736)], [(#734)].

Reviewer: ```suggestion
Dist-git patches are newly applied with `-p1` [(#730)]. Packit specfile code was improved [(#736)], [(#734)].
```

Author: fixed

Fedora-32 was added to 'fedora-all' and 'fedora-development' aliases [(#731)].
Documentation was improved by adding links to user configuration [(#726)].

Packit-service was adjusted, so we don't need metadata for copr build [(#431)], ansible deployment playbooks

> were adjusted, to use same playbooks in zuul and localy [(#429)]. 

Reviewer 2: ```suggestion
Packit-service was adjusted, so we don't need metadata for copr build [(#431)].
Ansible playbooks in deployment repo were adjusted to use the same playbooks in Zuul and locally [(#429)].
```

Author: fixed

> Build-helper classes were refactored [(#424)].

Reviewer: We can maybe merge the sentences about refactoring.

Author: fixed

> Bug related to incorrect pull-request filtering fixed [(#441)]

Reviewer: ```suggestion
Bug related to incorrect pull-request filtering fixed [(#441)].
```

Author: fixed

> Bug related to incorrect pull-request filtering fixed [(#441)]

Reviewer 2: ```suggestion
Bug related to incorrect pull-request filtering was fixed [(#441)].
```

Author: fixed

[(#730)]: https://github.com/packit-service/packit/pull/730
[(#736)]: https://github.com/packit-service/packit/pull/736
[(#734)]: https://github.com/packit-service/packit/pull/734
[(#731)]: https://github.com/packit-service/packit/pull/731
[(#726)]: https://github.com/packit-service/packit/pull/726
[(#431)]: https://github.com/packit-service/packit-service/pull/431
[(#429)]: https://github.com/packit-service/packit-service/pull/429
[(#426)]: https://github.com/packit-service/packit-service/pull/426
[(#424)]: https://github.com/packit-service/packit-service/pull/424
[(#441)]: https://github.com/packit-service/packit-service/pull/441

> 

Reviewer 3: please remove these
